src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -28,8 +28,16 @@ export const getAnalyticsSessionId = () => {
   return window.sessionStorage.getItem(INTEGRATIONS_ANALYTICS_SESSION_KEY);
 };
 
-export const isIntegrationDirectoryActive = () =>
-  localStorage.getItem(USE_INTEGRATION_DIRECTORY) === '1';
+export const isIntegrationDirectoryActive = (organization: Organization | undefined) => {
```

Comment:
@NisanthanNanthakumar Nit: You can always do this for potentially undefined parameters: `(organization?: Organization)`